server/validate/email/validate.go

```diff
@@ -224,13 +224,7 @@ func (v *validator) ResetSecret(email, scheme, lang string, tmpToken []byte, par
 	body := new(bytes.Buffer)
 	var login string
 	if params != nil {
-		var ok bool
-		login, ok = params["login"].(string)
-		if !ok || login == "" {
-			login = "-"
-		}
-	} else {
-		login = "-"
+		login = params["login"].(string)
```

Comment:
Maybe it's a good idea to panic if `params["login"]` is not a string. If so, please add a comment to that extent. If not add a `_`:`login, _ = params["login"].(string)`

Reply:
Added a comment.